packages/angular/package.json

```diff
@@ -90,7 +90,7 @@
     "karma-jasmine-html-reporter": "~2.1.0",
     "ng-packagr": "^17.3.0",
     "rxjs": "~7.8.1",
-    "storybook": "^8.0.10",
+    "storybook": "^8.1.1",
```

Comment:
Are we going to add tags to Angular wrappers? Or it's rather a separated category, so I'm not sure how we should treat those elements? :)

<img width="284" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/3519b898-61f4-48fa-8b84-77aa22fbe81b">

Reply:
I think the angular wrappers should reflect the web components they're based on.